AUTHORS

```diff
@@ -14,7 +14,7 @@ Contributors:
   Heikki Junnila, bug fixes for the debian packaging files
   Laurent (Renzo), Debian packages, FreeBSD & RDM testing.
   Lukas Erlinghagen, win32 port.
-  Johan Nilsson, Philips Hue trigger config
+  Johan Nilsson, Philips Hue & crelay trigger config
```

Comment:
Up to you, but you may want to change the Philips Hue bit to include Osram Lightify too now?